test/components/SingleDatePickerInput_spec.jsx

```diff
@@ -21,41 +21,45 @@ describe('SingleDatePickerInput', () => {
   describe('clear date', () => {
     describe('props.showClearDate is falsey', () => {
       it('does not have .SingleDatePickerInput__clear-date class', () => {
-        const wrapper = shallow(<SingleDatePickerInput showClearDate={false} />);
+        const wrapper = shallow(<SingleDatePickerInput id="date" showClearDate={false} />);
```

Comment:
All the changes here were just to clean up some prop type warnings that were bugging me.  Hope that's ok!